library/forms.inc

```diff
@@ -102,22 +85,18 @@ function addForm(
         $sql .= "?";
                 array_push($arraySqlBind, $date);
     }
-
     $sql .= ", ?, ?, ?, ?, ?, ?, ?, ?, ?)";
         array_push($arraySqlBind, $encounter, $form_name, $form_id, $pid, $user, $group, $authorized, $formdir, $therapy_group);
     return sqlInsert($sql, $arraySqlBind);
 }
-
 function authorizeForm($id, $authorized = "1")
 {
-    sqlQuery("update forms set authorized = '$authorized' where id = '$id' and deleted = 0");
+    $sql = ("UPDATE forms SET authorized = ? WHERE id = ? AND deleted = 0");
+    return sqlQuery($sql,array($forms,$id));
```

Comment:
Should be $authorized instead of $forms:
```
return sqlQuery($sql, [$authorized, $id]);
```

Reply:
Actually, should not be returning this, it should just be:
```
sqlQuery("update forms set authorized = ? where id = ? and deleted = 0", [$authorized, $id]);
```

Reply:
Thanks, I will review all this and give it a look, Thanks :)